module/Core/src/Core/Repository/AbstractRepository.php

```diff
@@ -3,10 +3,11 @@
 namespace Core\Repository;
 
 use Core\Entity\EntityInterface;
-use \Doctrine\ODM\MongoDB as ODM;
```

Comment:
I guess, Eclipse did not resolve this alias because of the backslash at the beginning, which is WRONG syntax..
I'm curious whether Eclipse accepted it without the leading backslash...